util/util.c

```diff
@@ -0,0 +1,317 @@
+// SPDX-License-Identifier: GPL-2.0
+
```

Comment:
Should this be GPL-2.0-or-later so a full header would look like this  
```
// This file is part of dracut.
// SPDX-License-Identifier: GPL-2.0-or-later
```
or is the C code GPL-2.0 only?

Reply:
yeah, the argument parsing code is copied from the kernel